odoo/addons/base/models/ir_ui_view.py

```diff
@@ -1193,15 +1200,16 @@ def _validate_tag_img(self, node, name_manager, node_info):
         if not any(node.get(alt) for alt in self._att_list('alt')):
             src = next((node.get(src) for src in self._att_list('src') if node.get(src)), "")
             if src:
+                # VFE FIXME this str construction looks strange
```

Comment:
do you want to keep this one?

Reply:
I wanted to ask @Xavier-Do POV :thinking:.
I suppose a a % (value) is probably missing here but maybe I'm missing something ^^.
`src = ' with src (%s)' % src`

Reply:
Oh right, indeed, looks a bit weird and we should try to avoid constructing sentences like this. Maybe add the whole `img` tag instead of just the `src` (`src` is not always a good indicator)

Reply:
I have to admit this is not correct (missing "% src") and this is not working with translation. You can just remove the src part to KISS since it wasn't working anyway. Don't know if the whole img tag is a good idea but also works for me, as you want.